service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -115,13 +119,104 @@ public class EhrExtractStatusService {
     private static final String FILENAME_TYPE_PLACEHOLDER = "FILENAME_PLACEHOLDER_ID=";
     private static final String ACKS_SET = ACK_HISTORY + DOT + ACKS;
     public static final int EHR_EXTRACT_SENT_DAYS_LIMIT = 8;
-    private static final String REASON_ERROR_CODE = "04";
-    public static final String REASON_ERROR_MESSAGE = "The acknowledgement has been received after 8 days";
+    private static final String UNEXPECTED_CONDITION_ERROR_CODE = "99";
```

Comment:
For a separate PR - it could be good to see if we can centralise these codes somehow, even via an enumeration.

Reply:
yes, I suggestions, I've taken a note of it and address it in a separate follow-up PR